ribasim_qgis/pyproject.toml

```
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ribasim_qgis"
+description = "Ribasim QGIS Plugin"
+readme = "README.md"
+authors = [{ name = "Deltares and contributors", email = "ribasim.info@deltares.nl" }]
+license = { text = "MIT" }
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Hydrology",
+]
+requires-python = ">=3.10"
```

```suggestion
requires-python = ">=3.9"
```

QGIS default installer still comes with Python 3.9, see also https://github.com/Deltares/Ribasim/pull/795.